### PR TITLE
jenkins/java-backend/Jenkinsfile - update with latest implemented steps

### PR DESCRIPTION
**Pull request by itewk**

### CODE REVIEW

File: jenkins/java-backend/Jenkinsfile [hunk 1]
```diff
@@ -50,10 +68,12 @@ spec:
     stage('Tag Source Code with Metadata') {
       steps {
         container('maven') {
-          sh '''
-            source tssc/bin/activate
-            python -m tssc -c cicd/tssc-config.yml -s tag-source
-          '''
+          withCredentials([usernamePassword(credentialsId: 'spo-jenkins-test-kjf', passwordVariable: 'GIT_PASSWORD', usernameVariable: 'GIT_USERNAME')]) {
```
Comment (HunterGerlach) [inline jenkins/java-backend/Jenkinsfile:71]: Is this the correct credentialsId val or does it need to be update? @itewk @kfrankli

Comment (kfrankli) [reply jenkins/java-backend/Jenkinsfile:71]: should probably fix that at some point, but that is the cred in jenkins currently.

Comment (kfrankli) [reply jenkins/java-backend/Jenkinsfile:71]: Can you make a JIRA? This also effects gitea.

Comment (HunterGerlach) [reply jenkins/java-backend/Jenkinsfile:71]: Ok, thanks for confirmation. Bug reported: https://projects.engineering.redhat.com/browse/NAPSSPO-604

**Pull request merged**